wiki/en/Unregistered-Servers.md

```diff
@@ -29,7 +29,7 @@ You should connect yourself using the local network address of the computer the
 
 ### Dynamic DNS and why you will probably need it
 
-Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a static address you can share with others. A "dynamic DNS" provider can supply you with a static IP address that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
+Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a static address you can share with others. A "dynamic DNS" provider can supply you with a static domain name that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
```

Comment:
"static domain name"?  I'm not familiar with the term.

> The Domain Name Service (DNS) maps hostnames to IP addresses. Dynamic DNS (DDNS) services automatically update their records as IP addresses change to ensure that clients requesting the record for a hostname always receive the correct IP address.

The _domain name_ is _fixed_ (as domain name always are) whilst the IP address is either _static_ or _dynamic_.  "Dynamic DNS" means using a short TTL (time to live) on the DNS A rec entry, so that the IP address for the name is checked often and, if the IP address for that name has changed, then the new IP address will be picked up.